### PR TITLE
fix(env-docs): removed auth variables and improved docs

### PR DESCRIPTION
**Pull request by crisgarner**
- Removed nextauth variables
- Changed default values on env.example
- Added video tutorial

### CODE REVIEW

**Review by kittybest — COMMENTED** [review]
I think some of the env parts overlapped with this PR: https://github.com/privacy-scaling-explorations/maci-rpgf/pull/163

How do you think about combining the content of this 2 PR?

**Comment by crisgarner** [comment]
> I think some of the env parts overlapped with this PR: #163
> 
> How do you think about combining the content of this 2 PR?

I agree, this branch is more updated so will add the contents of #163 to this one

**Comment by crisgarner** [comment]
Added and improved https://github.com/privacy-scaling-explorations/maci-rpgf/pull/163

File: docs/01_setup.md [hunk 1]
```diff
@@ -133,6 +105,67 @@ pnpm deploy-poll:NETWORK
...
+
+#### Network
+
+The default configuration is Optimism Sepolia for development and Optimism for production.
+
+You can find supported networks on the EAS documentation website: https://docs.attest.sh/docs/quick--start/contracts
+
+#### App
+
+Configure how many votes each voter receives and how many votes each project can receive.
```
Comment (ctrlc03) [inline docs/01_setup.md:149]: not sure about this part "how many votes each project can receive" - that's up to the voter, they could spend max credits on one single project. Perhaps you meant something else?

File: docs/01_setup.md [hunk 2]
```diff
@@ -133,6 +105,67 @@ pnpm deploy-poll:NETWORK
...
+You can also find configurations for when voting starts and ends as well as the registration and review period.
+
+Here, you can also configure who your admins are. These are the users who will approve applications and voters.
+
+To create your own round you need to do a few things:
+
+- Update `NEXT_PUBLIC_ADMIN_ADDRESS` a wallet address that approve the applications and voters (badgeholders)
```
Comment (ctrlc03) [inline docs/01_setup.md:156]: nit: approves

File: docs/01_setup.md [hunk 3]
```diff
@@ -133,6 +105,67 @@ pnpm deploy-poll:NETWORK
...
+- Set `NEXT_PUBLIC_ROUND_ID` to a unique identifier that will group the applications you want to list
+- Set `NEXT_PUBLIC_MACI_ADDRESS` - your deployed maci contract
+- Set `NEXT_PUBLIC_MACI_START_BLOCK` - block where your maci contract is deployed (optional)
```
Comment (ctrlc03) [inline docs/01_setup.md:159]: nit: optional but very much recommended (as it's a fallback to subgraph not setup/working)

File: docs/06_maci_troubleshooting.md [hunk 4]
```diff
@@ -23,7 +23,7 @@ There's no description in detail for signing up reverted by the contract. The fo
 
 If you found it displays "Voting is over", but you actually just started a poll:
 
-- Make sure that you are deploying through **MACI v1.2.2**.
+- Make sure that you are deploying through **MACI v1.2.4**.
```
Comment (ctrlc03) [inline docs/06_maci_troubleshooting.md:26]: nit: using rather than through

File: README.md [hunk 5]
```diff
@@ -12,6 +14,12 @@
 
 MACI-RPGF uses EAS as backbone to run Retroactive Public Goods Funding to reward contributors ([As used by the Optimism Collective](https://community.optimism.io/docs/governance/citizens-house/#how-retro-funding-works)) while adding a privacy layer to reduce bribery and collusion using MACI.
 
+## Video Tutorial
+
+A complete installation tutorial can be seen here:
+
+[![Watch the Video](https://img.youtube.com/vi/86VBbO1E4Vk/0.jpg)](https://www.youtube.com/watch?v=86VBbO1E4Vk)
```
Comment (ctrlc03) [inline README.md:21]: 🚀

**Review by ctrlc03 — CHANGES_REQUESTED** [review]
Thanks, left just a couple of comments

File: docs/01_setup.md (hunk 1 above)
Comment (kittybest) [reply docs/01_setup.md:149]: inside `.env.example` there are parameters as below:
```
# How many votes each ballot can hold
NEXT_PUBLIC_MAX_VOTES_TOTAL=150
# How many votes each project can receive
NEXT_PUBLIC_MAX_VOTES_PROJECT=50
```

thinking about removing this part also in this PR? or we could open another PR for removing this?

Comment (ctrlc03) [reply docs/01_setup.md:149]: I'll remove here

File: .gitignore [hunk 6]
```diff
@@ -38,3 +38,5 @@ deployed-contracts.json
 
 # typescript
 *.tsbuildinfo
+zkeys
+zkeys/**
```
Comment (0xmad) [inline .gitignore:42]: Empty line at the end

File: pnpm-lock.yaml
Comment (0xmad) [inline pnpm-lock.yaml:1]: This file shouldn't be changed

File: public/mockServiceWorker.js [hunk 7]
```diff
@@ -1,11 +1,14 @@
+/* eslint-disable */
+/* tslint:disable */
+
```
Comment (0xmad) [inline public/mockServiceWorker.js:3]: It's already in eslint ignore file, so no need to add these lines

**Review by 0xmad — COMMENTED** [review]
@crisgarner @ctrlc03 thanks!

**Review by 0xmad — APPROVED** [review]

**Pull request merged**